### PR TITLE
Diverge AT 12.0 from next

### PR DESCRIPTION
**Pull request by tuliom**
This patchset has the initial steps to diverge AT 12.0 from next.

It has already diverged the following packages:
 - gcc
 - glibc
 - binutils
 - OpenSSL
 - GDB

All the other packages still have to be reviewed and updated accordingly.

### CODE REVIEW

File: configs/next/build.mk [hunk 1]
```diff
@@ -0,0 +1 @@
+../11.0/build.mk
```
Comment (gftg85) [inline configs/next/build.mk]: Why not point to ../12.0/build.mk?

**Comment by gftg85** [comment]
OK with the changes.

File: configs/next/build.mk (hunk 1 above)
Comment (tuliom) [reply configs/next/build.mk]: It should.... It seems `make clone` has a bug.

**Comment by tuliom** [comment]
> Did you mean 12.0?

@gftg85 No.  AT next does indeed have version 13.0.  Don't you agree?

**Comment by tuliom** [comment]
@gftg85 Fixed the symlink pointing to AT 11.0.

**Comment by gftg85** [comment]
>AT next does indeed have version 13.0. Don't you agree?
I see what you mean, now.  Maybe the use of the word "Point" in your sentence is a bit misleading (since it could give the impression that it refers to symlinks, which point to 12.0), but not a big deal and fine the way it is.

**Pull request merged**